properties/P000025.md

```diff
@@ -1,11 +1,15 @@
 ---
 uid: P000025
-slug: sigma-locally-compact
-name: "$\\sigma$-Locally Compact"
+slug: sigma-neighborhood-compact
+name: "$\\sigma$-Neighborhood Compact"
+aliases:
+  - "$\\sigma$-Locally Compact"
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-A space is $\sigma$-locally compact if and only if it is $\sigma$-compact and locally compact.
+A space that is [$\\sigma$-compact](/properties/P000017.md) and [weakly locally compact](/properties/P000023.md).
```

Comment:
I think it should accept the syntax {P17}, {P23} with single braces.  See for example the last line in P000024.md.